src/config.py

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Is it possible to move the config file to the parent directory so that users don't have to go into the src directory, change setting, go to parent directory, then start the program with python3 src? It also makes it easier to manually access the config file since we don't have a gui for it yet.

Reply:
The configs in `config.py` are the default; if we want we can set up the env var so that if the user wants to override the defaults they can put their own configuration in a config folder somewhere (like `~/.config/loft/config.py` or something like that).